apps/client/src/pages/ConclusionPage.js

```diff
@@ -34,7 +34,7 @@ const ConclusionPage = ({ topic, checker }) => {
     houseNumberFull,
     postalCode,
     residence,
-  } = context.address;
+  } = sessionContext.address;
```

Comment:
Again, not sure why/when address is an map with an address per topic vs just the address itself.

Reply:
This was actually a bug. It needed `address[slug]`